src/modules.js/showDb.js

```diff
@@ -0,0 +1,40 @@
+export const showDb = (db) => {
+  const container = document.getElementById('list-item-container');
+
+  container.innerHTML = '';
+
+  db.forEach((element, index) => {
```

Comment:
- [ ] While you made great effort addressing and ensuring that the completion is controlled and saved to the local storage, kindly note that from the requirement is that clicking on the `clear completed` button should erase all the completed tasks in the array, meaning all the objects that has it's `completed` key value as `true`.  You can use `Array.filter()` higher order function for the functionality. 🙃

Reply:
fixed